### DISCUSSION TITLE
[2022/01/11] Frontend Night

### DISCUSSION THREAD

**Discussion by jjenko** [wantedly/frontend-night#187] [Archive]
## Policy

- 資料の準備をがんばらない
- 参加・途中退席は自由
- スキップしない

話したい・聞きたいネタを書いてく ✏️

ハッシュタグ [#frontend_night](https://twitter.com/hashtag/frontend_night?src=hash)

Prev: https://github.com/wantedly/frontend-night/discussions/186 

 _Created from https://github.com/wantedly/frontend-night/discussions/179 by [issue-creator](https://github.com/rerost/issue-creator)_

**Comment by NotFounds** [comment]
## 結局useMemoはいつ使えばいいの？　僕の決定版 - Qiita / @.uhyo
https://qiita.com/uhyo/items/5258e04aba380531455a

### 関連
- [そこのお前！　余計なuseMemo1個に含まれるオーバーヘッドは余計なdiv 0.57個分だぜ！ - Qiita](https://qiita.com/uhyo/items/59124425ca1dee3da891)
- [useMemoのコストを心配する前に余計なdivを減らせ！- Zenn](https://zenn.dev/uhyo/articles/usememo-time-cost)

---
- 基本的には自分たちもこの方針で実装していけば良さそう？
